core/src/data/schema.rs

```diff
@@ -2,16 +2,17 @@ use {
     crate::ast::{ColumnDef, ColumnOption, ColumnOptionDef, Expr},
     serde::{Deserialize, Serialize},
     std::fmt::Debug,
+    strum_macros::Display,
 };
 
-#[derive(Clone, Copy, Debug, Serialize, Deserialize)]
+#[derive(Clone, Copy, Debug, Serialize, Deserialize, PartialEq, Display)]
 pub enum SchemaIndexOrd {
     Asc,
     Desc,
     Both,
 }
```

Comment:
When we use string representation for SQL keyword, we use uppercased letters.
e.g.
<img width="382" alt="image" src="https://user-images.githubusercontent.com/2025065/169540953-038ce587-0368-46f6-b31e-511a0e83eaaf.png">

Could we have same patterns?
```rust
#[strum(to_string = "ASC")]
Asc,
```